org.freecadweb.FreeCAD.yaml

```diff
@@ -668,19 +668,25 @@ modules:
       - type: script
         dest-filename: FreeCAD.sh
         commands:
-          - exec /app/freecad/bin/FreeCAD -u ${XDG_CONFIG_HOME}/FreeCAD/user.conf -s ${XDG_CONFIG_HOME}/FreeCAD/system.cfg "$@"
+          - exec /app/freecad/bin/FreeCAD "$@"
```

Comment:
so will freecad find the user config in the right place? This is meant to have FreeCAD use `XDG_CONFIG_HOME`

(it's probably from the original flatpak that I didn't do)

Reply:
Sorry forgot to explain this, yes FreeCAD started following the xdg spec during 0.20 development, you will notice that the 0.19 flatpak installs add-ons ~./FreeCAD but 0.20 in a directory in the flatpak's data directory

Reply:
Now that I think about it I don't see why we would need a sh file anymore, could just export the executable directly